src/SelectMulti.vue

```diff
@@ -195,8 +199,12 @@
 				 * emits the most recently removed value,
 				 * *generally not necessary*, if state can be handled w/ v-model alone
 				 */
-				this.$emit('remove', this.selectedOptions[index])
+				const option = this.selectedOptions[index]
+				this.notificationMesasage = `option ${option.label} removed`
+				this.$emit('remove', option)
 				this.selectedOptions = [...this.selectedOptions.filter((_, i) => i !== index)]
+
+				setTimeout(() => { this.notificationMesasage = '' }, 50)
```

Comment:
Hi @RED143 - thanks for taking this on. I understand that some version of this may be required to ensure that the screen reader has a chance to pick up the message. Just because I worry about `setTimeout` a bit as a potential code smell, is there any chance `this.$nextTick()` could also work here? If it works just as well, it'd be nice to use that to have it captured by the Vue lifecycle. Otherwise, we can stick w/ `setTimeout()`

Reply:
I've tried to use a nextTick, but unfortunately this method doesn't work. I remember that we have similar cases with setTimeout in gamma.

Reply:
Sounds good, thanks @RED143